core/src/core/hash.rs

```diff
@@ -47,7 +47,10 @@ impl fmt::Debug for Hash {
 
 impl fmt::Display for Hash {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-		fmt::Debug::fmt(self, f)
```

Comment:
Want to do this in `Debug::fmt()` as well in case we're logging it this way anywhere?

Reply:
Should we actually log a full hash hex string since we're not limited in number of characters on screen in this case?

Reply:
I suspect logging the full hex string will make the logs significantly harder to read as they will start wrapping lines.

Reply:
Understood.